runtime/src/bank/mod.rs

```diff
@@ -556,16 +556,25 @@ impl Bank {
         self.store_account(pubkey, &new_account);
     }
 
+    fn inherit_sysvar_account_balance(&self, old_account: &Option<Account>) -> u64 {
+        // Corrent sysvar account balance maintenance activates at this epoch on the mainnet-beta
+        if self.epoch() >= 25 {
```

Comment:
Let's land this first than [`account.owner` hashing](https://github.com/solana-labs/solana/pull/9937/files#r422099886) considering significace: 

```
$ pry
[2] pry(main)> Time.now + ((432000 - 336972) / 2.5) + 4 * 3600 * 24
=> 2020-05-13 07:32:39 +0900
$ solana  --url http://api.mainnet-beta.solana.com epoch-info

Slot: 9840972
Epoch: 22
Epoch Slot Range: [9504000..9936000)
Epoch Completed Percent: 78.003%
Epoch Completed Slots: 336972/432000 (95028 remaining)
Epoch Completed Time: 1day 13h 26m 28s/2days (10h 33m 31s remaining)
```